### PR TITLE
[New] Add input-autogrow v1.1.0 with git auto-update

### PR DESCRIPTION
**Pull request by holtkamp**
Pull request for issue: https://github.com/cdnjs/cdnjs/issues/12644
Related issue(s): https://github.com/westonganger/input-autogrow/issues/7

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/westonganger/input-autogrow
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/input-autogrow
 * License and its reference: MIT, https://github.com/westonganger/input-autogrow/blob/master/LICENSE
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 1
   - Count of stars: 6
   - Count of forks: 1
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week: 669
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] Doesn't already exist / has duplicate issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
 * [x] The parent of the commit(s) in the PR is not older than 3 days.
 * [x] Pull request is sent from a non-master branch with a meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by extend1994** [comment]
@PeterDaveHello Does this lib have enough popularity to be added?

**Comment by holtkamp** [comment]
@extend1994 it is a better / maintained fork of https://github.com/Pixabay/jQuery-autoGrowInput

which has more stars, but is not maintained

**Comment by extend1994** [comment]
@holtkamp Thanks, I found it has enough popularity on npm.

File: ajax/libs/input-autogrow/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/westonganger/input-autogrow.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      }
```
Comment (extend1994) [inline ajax/libs/input-autogrow/package.json]: Let's also include the source file with the config
```js
      {
        "basePath": "src",
        "files": [
          "input-autogrow*"
        ]
      }
```

thanks.

Comment (holtkamp) [reply ajax/libs/input-autogrow/package.json]: @extend1994 mmm, ok, can you do this? Getting my fork in sync with the upstream master is a big challenge with this big repository. Takes hours for me to perform such a small change...

**Comment by extend1994** [comment]
Will try it later to pass the test.

**Comment by PeterDaveHello** [comment]
It's a DNS issue, should be fixed now.

**Comment by holtkamp** [comment]
@extend1994 @PeterDaveHello minor bump on this? Seems to be ready for merge...

File: ajax/libs/input-autogrow/package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  "description": "Autogrow plugin for inputs",
+  "license": "MIT",
+  "keywords": [
+    "autogrow",
+    "auto-grow",
+    "input",
+    "jquery",
+    "zepto",
+    "javascript",
+    "js"
```
Comment (sufuf3) [inline ajax/libs/input-autogrow/package.json]: Let's add more keywords to let people search this library more easier.
Like `autosize`, `html-forms`.
What do you think?
Many thanks.

screen shot 2018-05-07 at 12 42 25 am

Comment (westonganger) [reply ajax/libs/input-autogrow/package.json]: I'd happily accept a PR in `input-autogrow` for more/better keywords.

**Comment by holtkamp** [comment]
🎉

**Pull request merged**